src/I18n/Time.php

```diff
@@ -641,12 +641,29 @@ public function __toString()
      *   Or one of DateTimeZone class constants
      * @param string $country A two-letter ISO 3166-1 compatible country code.
      *   This option is only used when $filter is set to DateTimeZone::PER_COUNTRY
-     * @param bool $group If true (default value) groups the identifiers list by primary region
+     * @param bool|array $options If true (default value) groups the identifiers list by primary region.
+     *   Otherwise, an array containing `group`, `abbr`, `before`, and `after`
+     *   keys. Setting `group` and `abbr` to true will group results and append
+     *   timezone abbreviation in the display value. Set `before` and `after`
+     *   to customize the abbreviation wrapper.
      * @return array List of timezone identifiers
      * @since 2.2
      */
-    public static function listTimezones($filter = null, $country = null, $group = true)
+    public static function listTimezones($filter = null, $country = null, $options = [])
     {
+        if (is_bool($options)) {
+            $options = [
+                'group' => $options,
+            ];
+        }
+        $options = array_merge([
+            'group' => true,
+            'abbr' => false,
+            'before' => ' - ',
+            'after' => null,
+        ], $options);
```

Comment:
Cool. I'll keep that in mind for the future.

Can you merge/squash your commit onto this branch, or close this and have yours supersede this.